### PR TITLE
Upgrade deps

### PR DESCRIPTION
**Pull request by raymondfeng**
Fixes the `npm audit` complaint reported by https://github.com/strongloop/loopback4-example-shopping/issues/8

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -1 +1 @@
-package-lock=false
+package-lock=true
```
Comment (virkt25) [inline .npmrc]: Can we just get rid of this file as the default behaviour is to create a lock file?

Comment (raymondfeng) [reply .npmrc]: Some developers (like me) have a global npm config to disable `package-lock`.

File: package.json [hunk 2]
```diff
@@ -52,32 +52,32 @@
...
   "devDependencies": {
-    "@commitlint/cli": "^7.0.0",
-    "@commitlint/config-conventional": "^7.0.1",
-    "@commitlint/travis-cli": "^7.0.0",
-    "@loopback/build": "^0.7.0",
-    "@loopback/testlab": "^0.11.2",
+    "@commitlint/cli": "^7.1.2",
+    "@commitlint/config-conventional": "^7.1.2",
+    "@commitlint/travis-cli": "^7.1.2",
+    "@loopback/build": "^0.7.2",
```
Comment (virkt25) [inline package.json]: 0.7.3

File: package.json [hunk 3]
```diff
@@ -52,32 +52,32 @@
...
+    "@loopback/testlab": "^0.12.1",
```
Comment (virkt25) [inline package.json]: 0.12.2

**Comment by raymondfeng** [comment]
Close to favor #10

**Pull request closed without merging**